extensions/keycloak-admin-rest-client/runtime/pom.xml

```diff
@@ -22,16 +22,6 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-tls-registry</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.sun.activation</groupId>
-                    <artifactId>jakarta.activation</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
```

Comment:
Same here actually. We now have a dependency with the Apache HTTP Client and I don't think we want (except if I'm mistaken) that.

Reply:
@gsmet This is a mistake at `keycloak-client-common-synced` side. I will fix it there and update this PR with new version of `keycloak-client`.

Reply:
Thanks I will have another look next week.

Reply:
@pskopek I think I misunderstood you. The issue is not fixed, right?

Reply:
I released new version of keycloak-client (26.0.3) and updated the PR. It is fixed now.